recipe/meta.yaml

```
@@ -23,7 +23,7 @@ requirements:
   run:
     - python
     - h5py
-    - hdf5-lz4
+    - hdf5-external-filter-plugins
```

This is to coordinate the changes with https://github.com/nsls-ii-forge/analysis-feedstock/pull/41:
```suggestion
    - hdf5-external-filter-plugins-bitshuffle
    - hdf5-external-filter-plugins-lz4
```